src/lib/snarkette/bg.ml

```diff
@@ -0,0 +1,193 @@
+open Core_kernel
```

Comment:
I think you didn't mean to share this right?

Reply:
removed :). sorry.